src/commands/config.rs

```diff
@@ -33,3 +57,21 @@ pub fn global_config(email: &str, api_key: &str) -> Result<(), failure::Error> {
 
     Ok(())
 }
+
+#[cfg(all(test, not(target_os = "windows")))]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn it_creates_file_with_perms() {
+        let dir = env::temp_dir().join("wrangler_test_config_dir");
+        global_config("email", "api_key", Some(dir.clone())).expect("could not create file");
+        assert_eq!(dir.exists(), true);
+
+        let metadata = dir.metadata().unwrap();
+        let permissions = metadata.permissions();
+        assert_eq!(permissions.mode(), CONFIG_PERM);
```

Comment:
Currently this fails with:
```
  left: `16893`,
 right: `256`', src/commands/config.rs:74:9
```

It's 40755 in octal which is not what I set :man_shrugging:

Reply:
That said, It seems that permissions don't apply.